libraries/AP_NavEKF3/AP_NavEKF3_PosVelFusion.cpp

```diff
@@ -589,12 +590,9 @@ void NavEKF3_core::SelectVelPosFusion()
 void NavEKF3_core::FuseVelPosNED()
 {
     // health is set bad until test passed
-    bool velHealth = false; // boolean true if velocity measurements have passed innovation consistency check
-    bool posHealth = false; // boolean true if position measurements have passed innovation consistency check
-    bool hgtHealth = false; // boolean true if height measurements have passed innovation consistency check
-
-    // declare variables used to check measurement errors
-    Vector3f velInnov;
+    bool velCheckPassed = false; // boolean true if velocity measurements have passed innovation consistency checks
+    bool posCheckPassed = false; // boolean true if position measurements have passed innovation consistency check
```

Comment:
these 3 are not quite consistent. velCheckPassed and hgtCheckPassed are consistent, they are strictly based on passing the innovation check. The posCheckPassed value can be true even if innov check fails. Is that deliberate?

Reply:
It's an oversight - line 714 was removed and then made it back in when the changes were split into separate patches. It has now been removed.